Libraries/Components/Pressable/Pressable.js

```diff
@@ -180,15 +180,28 @@ function Pressable(props: Props, forwardedRef): React.Node {
       ? {...props.accessibilityState, disabled}
       : props.accessibilityState;
 
-  const restPropsWithDefaults: React.ElementConfig<typeof View> = {
+  let restPropsWithDefaults: React.ElementConfig<typeof View> = {
```

Comment:
Can you keep this as `const` and instead create a new `const` local variable that contains the ripple props (or remains undefined) that you spread in here?

It's a minor code reorganization, but it helps keep the code slightly more readable in the long-term the more you can have `const` variables that — as you read the declaration — you know will not be reassigned later.